libs/block-content/website/src/lib/facebook-video-block.tsx

```diff
@@ -24,6 +24,7 @@ export const FacebookVideoBlock = ({
       height={'auto'}
       url={`https://www.facebook.com/${userID}/videos/${videoID}/`}
       controls={true}
+      config={{facebook: {playerId: 'facebook-player'}}}
```

Comment:
this now causes that you can not have the multiple facebook videos as they have the same Id. 
Please combine this with `const id = useId()`